src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ItemsViewTests/ItemsViewSummaryPage.xaml.cs

```diff
@@ -18,8 +18,9 @@
 
 namespace UITests.Microsoft_UI_Xaml_Controls.ItemsViewTests;
 
-// Samples times out on iOS.
+#if DEBUG || !__IOS__ // Samples times out on iOS.
```

Comment:
This is now ignored in snapshot tests already

Reply:
Ok, I'll remove this in my next PR then ... don't want to trigger a new build for that :)